Automattic/jetpack
# [Earn] Make the Donation block available on Free plans

---





* Relates to https://github.com/Automattic/gold/issues/11
* To be tested with D108873-code
* Relates to https://github.com/Automattic/wpcomsh/pull/1358

## Proposed changes:

* Make Donation block free for all plans and charge a 10% commission for free blogs on free plans. `should_upgrade_to_access_memberships` is no longer needed as recurring payments will be free for all users

### Other information:


## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






1. Sync code to WPCOM
2. Sandbox your site, `subscribe.wordpress.com` and `public-api.wordpress.com`
3.  On a site with a free plan, add a Donation block to post and save it
4.  Open the post in an incognito window and verify the user can checkout successfully
